README.md

```diff
@@ -11,11 +11,15 @@ Workers](https://docs.improbable.io/reference/latest/shared/glossary#worker) for
 
 ## Dependencies
 
-This project requires [CMake](https://cmake.org/) to be installed. CMake is the build system used to build the project.
+This project requires the following software to be installed:
+- [CMake](https://cmake.org/) 3.0+
+- _(Windows)_ Visual Studio 2015 or above.
+- _(macOS)_ Xcode
+- _(Linux)_ `gcc` / `clang` and `make` (for the "Unix Makefiles" CMake project generator)
 
 ## Quick start
 
-The build scripts for each project:
+The build scripts (`build.json`) for each project:
```

Comment:
nit: Do we want to refer to these as being within each project, or each worker type?